src/ontology/uberon-edit.obo

```diff
@@ -2174,6 +2174,7 @@ def: "The surface (external) layer of ectoderm which begins to proliferate short
 subset: pheno_slim
 synonym: "surface (external) ectoderm" EXACT []
 synonym: "surface ectoderm" EXACT [MP:0012532]
+xref: BTO:0006242 {http://purl.org/dc/terms/contributor="https://orcid.org/orcid:0000-0003-4423-4370"} ! external ectoderm
```

Comment:
Until the OBO format has explicit support for arbitrary axiom annotations, annotations with unsupported annotation properties are best avoided. With the current OBO parser, they get mangled into monstrosities like this: `http://www.geneontology.org/formats/oboInOwl#http://purl.org/dc/terms/contributor`.

Please stick to `oboInOwl:hasDbXref` annotations for the time being.

And what is this ORCID format: `https://orcid.org/orcid:0000-0003-4423-4370`? Please use `https://orcid.org/0000-0003-4423-4370`.

Reply:
Thanks for pointing out the issue with the ORCID IRIs - that was an oversight.

However, I have seen arbitrary axiom annotations in other ontologies that curate their edit file in OBO format such as MONDO.

I have tested further and found that switching `http://purl.org/dc/terms/contributor` for `dcterms:contributor` works as intended. Here's the OWL axiom that's generated after converting from OBO format to OWL RDF/XML:

```xml
<owl:Axiom>
    <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/UBERON_0000018"/>
    <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
    <owl:annotatedTarget>IDOMAL:0002421</owl:annotatedTarget>
    <dcterms:contributor>https://orcid.org/0000-0003-4423-4370</dcterms:contributor>
</owl:Axiom>
```

Is there another issue with arbitrary axioms in the OBO format that I'm not aware of?

Reply:
> I have tested further and found that switching http://purl.org/dc/terms/contributor for dcterms:contributor works as intended.

No it doesn’t: it creates an annotation with a `dcterms:contributor` property but look at the `dcterms:` prefix declaration in the XML header:

```
xmlns:dcterms="http://www.geneontology.org/formats/oboInOwl#dcterms:"
```

This results in the annotation property effectively being `http://www.geneontology.org/formats/oboInOwl#dcterms:contributor`, which obviously is incorrect.

Please don’t try to circumvent the limitations of the OBO format unless you really know what you are doing.

Reply:
Is there a way to include an appropriate definition of DCTERMS? otherwise I can change this to use the oboInOwl relation

Reply:
Not with the current version of our tools. The next version of ROBOT should allow to use any annotation property you want, with the syntax you initially used (`{http://purl.org/dc/terms/contributor="..."}`).

Rumour has it that the next version of ROBOT is only a couple of weeks away from being released, so if you’d really like to use that property we can simply put that PR on hold until then.

Otherwise, for now you can use either `{source="..."}` (preferably) or `{hasDbXref="..."}` (_should_ work).

Reply:
I think I can wait for the next version of ROBOT! This will be great. Thanks for the clarification

Reply:
Is there a PR associated with this change in ROBOT we can link to?

Reply:
The real change is in the OWL API. It happened here: https://github.com/owlcs/owlapi/pull/1099 and is now available in OWL API version 4.5.26.

The change in ROBOT is simply to upgrade the OWL API from 4.5.25 to 4.5.26: https://github.com/ontodev/robot/pull/1135

Reply:
@gouttegd I updated to use the syntax you specified. With the new release of ROBOT, do you think this works now? Or are there other changes I should make?

Reply:
This will work, but we must still wait for the next ODK release (which will include the latest ROBOT version), _and_ for the next Protégé release (which will use OWL API 4.5.26). Both should happen next week.

One thing you I’d like to ask you to change in the meantime, though, is removing the comment at the end of each statement (e.g., ` ! external ectoderm`). It does not really serve any purpose, and more importantly it will be automatically removed anyway the first time any editor does any change with Protégé and saves the -edit file. That’s because Protégé has no way of knowing the label of the foreign term (I assume in your case it comes from Biomappings, but Protégé is ignorant of that – for Protégé the xref is opaque).

So in order to avoid an otherwise unavoidable spurious diff in the first commit after this PR is merged, please remove those comments.

Reply:
Done in 60f2ad6df.

Reply:
Excellent. As soon as we’ll have ODK 1.4.3 there should be no obstacle to merging this.